source/shared/cpp/ObjectModel/Util.cpp

```diff
@@ -75,41 +75,127 @@ std::string ValidateColor(const std::string& backgroundColor,
 }
 
 void ValidateUserInputForDimensionWithUnit(const std::string &unit, const std::string &requestedDimension,
-    int &parsedDimension)
+    int &parsedDimension, std::vector<std::shared_ptr<AdaptiveCardParseWarning>>& warnings)
 {
-    if (requestedDimension.empty())
+    int parsedVal = 0;
+    parsedDimension = 0;
```

Comment:
Do the renderers ignore the pixel width if it's 0?

Reply:
yes, for column and image renderer will ignore 0 val when both width & height are set to zeroes

---
In reply to: [216388662](https://github.com/Microsoft/AdaptiveCards/pull/1872#discussion_r216388662) [](ancestors = 216388662)